cli/src/pcluster/config/update_policy.py

```diff
@@ -296,6 +318,38 @@ def condition_checker_queue_update_strategy(change, patch):
     return result
 
 
+def condition_checker_resize_update_strategy_on_remove(change, patch):
+    # Check if fleet is stopped
+    result = not patch.cluster.has_running_capacity()
+
+    # Check if QueueUpdateStrategy is TERMINATE
+    if is_slurm_queues_change(change) or change.key == "SlurmQueues":
+        result = result or is_resize_update_strategy_set(patch)
+
+    # SlurmQueue or ComputeResource can be added
+    if change.is_list and (is_slurm_queues_change(change) or change.key == "SlurmQueues"):
+        result = result or (change.old_value is None and change.new_value is not None)
+
+    # Check if MaxCount is increased
+    if change.key == "MaxCount":
+        result = result or (int(change.new_value) if change.new_value is not None else DEFAULT_MAX_COUNT) >= (
+            int(change.old_value) if change.old_value is not None else DEFAULT_MAX_COUNT
+        )
+
+    # Check if MinCount is increased and MaxCount is increased of at least the same amount
```

Comment:
> I mean if I have MinCount=1 and MaxCount=10, can I simply increase MinCount to 2 without the need to increase MaxCount too?
 
no, because in your example the number of dynamic nodes will be decreased (hence terminated).

Reply:
thanks!